HackYourFuture-CPH/kp-toolbox-fp-class21
# Feature/fp21 81 contact form

# Description

## Frontend: 
Contact Us page with contact form component and message has been sent component, and page Inbox with incoming messages that will be visible only for an admin user. 
## Backend: 
Added routes for adding and retrieving data from the database (messages from users). Created migration for table user_enquiry where sent messages will be stored.
![Screenshot 2022-08-15 at 14 12 10](https://user-images.githubusercontent.com/92082885/184633051-0fd3cefb-6ccc-43d7-807c-9cbb23c6ca8e.png)
![Screenshot 2022-08-15 at 14 11 05](https://user-images.githubusercontent.com/92082885/184633059-73b087a4-4ddd-4a5c-a303-3ac2ee7d442e.png)
![Screenshot 2022-08-15 at 14 10 44](https://user-images.githubusercontent.com/92082885/184633063-c382a8e4-c370-498a-9330-71ca5a920ab8.png)

# How to test?

1. Run yarn db:clean and yarn db:setup to update migration files.
2. Run both server and client and go to the contact us page by pressing the link on the navigation panel.
3. To access the admin page's inbox, go directly to http://localhost:3000/inbox-admin.
4. Run storybook to see message has been sent component.

# Checklist
